### ISSUE TITLE
Address already in Use: Unable to Bind

### ISSUE THREAD

**Issue by Harbir123** [allinurl/goaccess#1007]
I run a command 

    sudo goaccess /var/log/apache2/access.log --log-format=COMBINED -a -o /var/www/html/report.html  --real-time-html

However, I am getting the following error over and over again. I tired restarting the apache server but still problem persists.

```
Fatal error has occurred
Error occured at: src/websocket.c - ws_socket - 2746
Unable to set bind: Address already in use.

```

**Comment by allinurl** [comment]
You can try to find which process is using the port and kill it. e.g.,

    lsof -i tcp:7890

Please also take a look at `--addr` and `--port` [options](https://goaccess.io/man#options) if you need to use different settings.

**Comment by allinurl** [comment]
Closing this. Feel free to reopen it if needed.

**Comment by Harbir123** [comment]
Thnks

On Feb 14, 2018 7:06 AM, "Gerardo O." <notifications@github.com> wrote:

Closed #1007 <https://github.com/allinurl/goaccess/issues/1007>.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/allinurl/goaccess/issues/1007#event-1472909958>, or mute
the thread
